### ISSUE TITLE
[cloud-tensorflow-mnist]: little sentence error

### ISSUE THREAD

**Issue by ghost** [googlecodelabs/feedback#510]
In the part 3. Theory: train a neural network of the tutorial it is written:

" You also see if the neural network, in its current state of training, has recognized them (white background) or mis-classified them (red background with **correct label in small print on the left side, bad computed label on the right of each digit**). "
By looking at the image it should be: correct label in small print on the RIGHT and bad computed label on the LEFT.

I hope i am not mistaken,
Thank you!

**Comment by martin-gorner** [comment]
Thank you for your feedback. The entire lab was revamped for modern Tensorflow and now uses Keras as its high-level API. I have changed the visualizations as well and bad recognitions are now displayed in red. No more left/right confusion.
